src/main/java/org/testobject/rest/api/model/TestToRun.java

```diff
@@ -0,0 +1,52 @@
+package org.testobject.rest.api.model;
+
+import java.util.Objects;
+
+public class TestToRun {
+
+	private final String testClass;
+	private final String testMethod;
+
+	public TestToRun(String testClass, String testMethod) {
+		this.testClass = testClass;
+		this.testMethod = testMethod;
+	}
+
+	public TestToRun(String testClass) {
+		this.testClass = testClass;
+		this.testMethod = null;
+	}
+
+	public String getTestClass() {
+		return testClass;
+	}
+
+	public String getTestMethod() {
+		return testMethod;
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o)
+			return true;
+		if (o == null || getClass() != o.getClass())
+			return false;
+		TestToRun testToRun = (TestToRun) o;
```

Comment:
Please name `testToRun` -> `other`. This way it is more clear what is being compared.